config/environments/development.php

```diff
@@ -19,3 +19,6 @@
 if (env('WP_OFFLOAD_MEDIA_PRESET')) {
     Config::define('WP_OFFLOAD_MEDIA_PRESET', env('WP_OFFLOAD_MEDIA_PRESET'));
 }
+
+const SENTRY_TRACES_SAMPLE_RATE = 1.0;
+const SENTRY_PROFILE_SAMPLE_RATE = 1.0;
```

Comment:
Should be 
```
Config::define('SENTRY_TRACES_SAMPLE_RATE', 1.0);
```
To prevent constant clashes in other files. Unless you'd like to change the values on different platforms I would put these in application.php - the catch all configuration (prod) file